pkg/api/handlers/libpod/images.go

```diff
@@ -311,7 +311,7 @@ func ImagesLoad(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
 	decoder := r.Context().Value("decoder").(*schema.Decoder)
 	query := struct {
-		Reference string `schema:"reference"`
+		Name string `schema:"name"`
```

Comment:
I prefer to not introduce breaking changes to the endpoints.  Renaming a parameter has IMHO little benefit to users but may break them.